_topic/android-restricted-settings.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Restricted settings on Android."
+prompt: "Learn how to update restricted settings on Android for the Pebble app to work properly on Android 13+."
+keywords: "android app settings notifications"
+date:   2023-01-03 06:53:00
+author: "BlockArchitech"
+hideFirstStepNumber: false
+osSpecific: true
```

Comment:
This should be 'false'. 
OSSpecific means that there is a separate Android/iOS guide and will prompt the user for their OS (like /setup does). As there's _only_ a guide for Android, please leave as false

Reply:
I'll update the help.rebble.io/contributing page to make it clearer that this should only be true when there's an iOS AND an Android set of instructions.